03-storage/20-notification.md

```diff
@@ -0,0 +1,13 @@
+## Notifications
+
+Storage nodes can produce notifications about internal events for external listeners. The specification covers only basic concepts of notifications and their triggers and does not define implementation details such as message binary format or transport protocol. Such details may vary in storage nodes depending on external listeners. Notifications do not affect core protocol and can be disabled. Therefore, one can use it only in a controlled environment with access to storage node configuration.
+
+Notification is the entity that consists of **topic** and **message**.
+
+### Object notifications
+
+Stored objects can trigger notifications. Object triggers notification if it contains valid well-known object header `__NEOFS__TICK_EPOCH`. Read more about well-known headers in the NeoFS API v2 section.
```

Comment:
Worth mentioning what happens if multiple `__NEOFS_TICK_EPOCH` attributes are present. Like am I limited by a single notification per object?

Reply:
There can't be multiple attributes with the same name, according to the spec.